SPECS/metal3-cluster-api-provider-metal3/README.md

```diff
@@ -0,0 +1,18 @@
+Adds the three packages for metal3 containers who components are written in
+golang:
```

Comment:
Nit: this first sentence is a little confusing. Makes it sound like all metal3 containers need these three packages. Maybe this is the case.

Reply:
Thanks for catching this. I will move the contents of this file to the individual spec files.